src/stateProviders/provideLoggedInUser.js

```diff
@@ -9,6 +12,24 @@ export default provideState({
     percentageFilled: 0,
   }),
   effects: {
+    initialize: effects => state => {
+      const { setToken, setUser } = effects;
+      const jwt = localStorage.getItem('EGO_JWT');
+      if (jwt) {
+        try {
+          const gapi = global.gapi;
```

Comment:
Couldn't find anything much better. But maybe we just do this right before logout if `gapi.auth` doesn't exist?